server/utils/jwt.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+const { SECRET_TOKEN } = process.env;
+
+const sign = (data) =>
+  new Promise((resolve, reject) => {
+    jwt.sign(data, SECRET_TOKEN, (err, decoded) => {
+      if (err) return reject(err);
+      return resolve(decoded);
+    });
+  });
```

Comment:
It's common to call it `payload` and it's `encoded`
```suggestion
const sign = (payload) =>
  new Promise((resolve, reject) => {
    jwt.sign(payload, SECRET_TOKEN, (err, encoded) => {
      if (err) return reject(err);
      else return resolve(encoded);
    });
  });
```